courseaffils/columbia.py

```diff
@@ -145,7 +145,7 @@ def to_string(cdict):
     @staticmethod
     def to_dict(wind_string):
         wind_match = re.match(
-            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\d{3})'
+            ('t(?P<term>\d).y(?P<year>\d{4}).s(?P<section>\w{3})'
```

Comment:
This change is in the WindTemplate regexp, but the test you added is for the SectionkeyTemplate. I'm not sure which one you actually intended to change.